samples/04.0-file-transfer/integration-tests/src/testFixtures/java/org/eclipse/dataspaceconnector/extension/sample/test/FileTransferSampleTestUtils.java

```diff
@@ -0,0 +1,207 @@
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial test implementation for sample
+ *
+ */
+
+package org.eclipse.dataspaceconnector.extension.sample.test;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.path.json.JsonPath;
+import org.apache.http.HttpStatus;
+import org.eclipse.dataspaceconnector.junit.testfixtures.TestUtils;
+import org.eclipse.dataspaceconnector.spi.types.domain.DataAddress;
+import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataRequest;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.Matchers.emptyString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+
+public class FileTransferSampleTestUtils {
+
+    static final ObjectMapper MAPPER = new ObjectMapper();
+
+    //region constant test settings
+    static final String INITIATE_CONTRACT_NEGOTIATION_URI = "http://localhost:9192/api/v1/data/contractnegotiations";
+    static final String LOOK_UP_CONTRACT_AGREEMENT_URI = "http://localhost:9192/api/v1/data/contractnegotiations/{id}";
+    static final String INITIATE_TRANSFER_PROCESS_URI = "http://localhost:9192/api/v1/data/transferprocess";
+    static final String CONTRACT_OFFER_FILE_PATH = "samples/04.0-file-transfer/contractoffer.json";
+    static final String TRANSFER_FILE_PATH = "samples/04.0-file-transfer/filetransfer.json";
+    static final String DESTINATION_FILE_PATH = "samples/requested.test.txt";
+    static final String API_KEY_HEADER_KEY = "X-Api-Key";
+    static final String API_KEY_HEADER_VALUE = "password";
+    static final File DESTINATION_FILE = getFileFromRelativePath(FileTransferSampleTestUtils.DESTINATION_FILE_PATH);
+    //endregion
+
+    //region changeable test settings
+    final String sampleAssetFilePath;
+    final File sampleAssetFile;
+    Duration timeout = Duration.ofSeconds(15);
+    Duration pollInterval = Duration.ofMillis(500);
+    //endregion
+
+    String contractNegotiationId;
+    String contractAgreementId;
+
+    FileTransferSampleTestUtils(@NotNull String sampleAssetFilePath) {
+        this.sampleAssetFilePath = sampleAssetFilePath;
+        this.sampleAssetFile  = getFileFromRelativePath(sampleAssetFilePath);
+    }
+
+    /**
+     * Assert that prerequisites are fulfilled before running the test.
+     * This assertion checks only whether the file to be copied is not existing already.
+     */
+    void assertTestPrerequisites() {
+        assertThat(DESTINATION_FILE).doesNotExist();
```

Comment:
same here as above destination file path can be taken as argument and caller will control over the name and path.

Reply:
I've made the destination file path configurable as discussed.